### ISSUE TITLE
Markbot issues

### ISSUE THREAD

**Issue by AveryMillss** [AveryMillss/content-semantics#1]
Markbot says 
line 60: " in attribute name. probable cause: matching quote missing somewhere earlier
line 60: element time="aug not allowed as child element small in this context
line 60: stray end tag time
line 60: end tag small violates nesting rules

not sure what to do. @thomasjbradley

**Comment by thomasjbradley** [comment]
It looks to me like you fixed that error. Is it still causing problems?

**Comment by AveryMillss** [comment]
Yes I fixed it! but the my other issues about the fix the html and fix the css are still issues!!!
